### PR TITLE
Adds Aparna as an author

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Adds Aparna as an author for blog #1398 

### Check List
- [ x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/apasun.markdown [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: Aparna Sundar
+short_name: apasun
+photo: '/assets/media/authors/apasun.jpg'
+github: apasun
+linkedin: aparnasundar
+---
+
+**Dr. Aparna Sundar** is a Senior UX Researcher at OpenSearch covering all areas of UI in the OpenSearch product. She has over 20 years of experience in the field of research and design and actively publishes in the area of cognitive science.
```
Comment (natebower) [inline _authors/apasun.markdown:9]:
```suggestion
**Dr. Aparna Sundar** is a senior UX researcher at AWS covering all areas of the OpenSearch UI. She has over 20 years of experience in the field of research and design and actively publishes in the area of cognitive science.
```

**Review by natebower — APPROVED** [review]
LGTM

**Comment by dtaivpp** [comment]
@kolchfa-aws can you fix the DCO on your most recent commit?

**Review by AMoo-Miki — APPROVED** [review]

**Comment by AMoo-Miki** [comment]
Set DCO to pass as the issue was with a pull+merge. The commits that included changes were appropriately signed.

**Pull request merged**